python/lsst/analysis/tools/interfaces/_task.py

```diff
@@ -212,7 +213,8 @@ class AnalysisBaseConfig(PipelineTaskConfig, pipelineConnections=AnalysisBaseCon
         doc="Which time stamp should be used as the reference timestamp for a "
         "metric in a time series database, valid values are; "
         "reference_package_timestamp, run_timestamp, current_timestamp, "
-        "and dataset_timestamp",
+        "dataset_timestamp and explicit_timestamp:datetime where datetime is"
+        "is given in the form %Y%m%dT%H%M%S%z",
```

Comment:
So, %z is the time zone in format "±HHMM". AFAICT, times uploaded to the Chronograf are in Zulu time, i.e., UTC+0, i.e., GMT. Would that equate to +0000 in this format?

Reply:
You can do +0000 or z which is an explicit carve out, I think because it is such a commonly output format